### ISSUE TITLE
Xray air blocks are showing up

### ISSUE THREAD

**Issue by IsaiahJM97** [MichaelHillcox/XRay-Mod#87]
**Describe the bug**
It seems that the air block is selected in the Xray. I have not changed any settings except turning the distance from 8 to 32 and turning on coal, and iron ores but the air block is on the Xray too.

**To Reproduce**
3. ...

**Expected behavior**
When I use the Xray, I do not expect to see the air block selected in the Xray

**Screenshots**
https://imgur.com/a/Gng5kWL
![image](https://user-images.githubusercontent.com/12649107/48937335-a0953800-eec2-11e8-8ed9-5d45bfc7a880.png)
![image](https://user-images.githubusercontent.com/12649107/48937349-aa1ea000-eec2-11e8-8d75-bf42a8551d65.png)
![image](https://user-images.githubusercontent.com/12649107/48937355-ad199080-eec2-11e8-9dcd-fccd85f5ddd5.png)
![image](https://user-images.githubusercontent.com/12649107/48937363-b0ad1780-eec2-11e8-8018-7d59a9da7cc7.png)

**Minecraft Enviorment**
 - Minecraft Version: [1.12.2]
 - XRay Mod Version: [1.5.0]
 - Forge Version if applicable [14.23.5.2775]
 

**Additional context**
Nothing

**Comment by MichaelHillcox** [comment]
Could you pastebin your config? also the likely hood is that if you delete your config this issue will go away :P

**Comment by IsaiahJM97** [comment]
I apologize for the long await. I can paste my config, but just so you know, this does happen with Gold ore as well. I also did try deleting my config and the issue went away the first time.

`# Configuration file

~CONFIG_VERSION: 1.4

general {
    I:searchdist=4
    B:show-overlay=true
}


ores {

    "minecraft:quartz_ore:0" {
        S:name=minecraft:quartz_ore
        I:meta=0
        I:red=30
        I:green=74
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:gold_ore:0" {
        S:name=minecraft:gold_ore
        I:meta=0
        I:red=255
        I:green=255
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:redstone_ore:0" {
        S:name=minecraft:redstone_ore
        I:meta=0
        I:red=255
        I:green=0
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "grimpack:uranium_ore:0" {
        S:name=grimpack:uranium_ore
        I:meta=0
        I:red=0
        I:green=255
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "grimpack:randomite:0" {
        S:name=grimpack:randomite
        I:meta=0
        I:red=201
        I:green=166
        I:blue=255
        B:enabled=true
        B:useoredict=false
    }

    "minecraft:iron_ore:0" {
        S:name=minecraft:iron_ore
        I:meta=0
        I:red=170
        I:green=117
        I:blue=37
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:emerald_ore:0" {
        S:name=minecraft:emerald_ore
        I:meta=0
        I:red=0
        I:green=136
        I:blue=10
        B:enabled=true
        B:useoredict=true
    }

    "tconstruct:ore:1" {
        S:name=tconstruct:ore
        I:meta=1
        I:red=255
        I:green=153
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:coal_ore:0" {
        S:name=minecraft:coal_ore
        I:meta=0
        I:red=0
        I:green=0
        I:blue=0
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:lapis_ore:0" {
        S:name=minecraft:lapis_ore
        I:meta=0
        I:red=0
        I:green=0
        I:blue=255
        B:enabled=true
        B:useoredict=true
    }

    "tconstruct:ore:0" {
        S:name=tconstruct:ore
        I:meta=0
        I:red=0
        I:green=0
        I:blue=255
        B:enabled=false
        B:useoredict=true
    }

    "minecraft:diamond_ore:0" {
        S:name=minecraft:diamond_ore
        I:meta=0
        I:red=136
        I:green=136
        I:blue=255
        B:enabled=false
        B:useoredict=true
    }

}


`
Or here is the cfg converted to a  .txt file
[xray.txt](https://github.com/MichaelHillcox/XRay-Mod/files/2624287/xray.txt)

**Comment by MichaelHillcox** [comment]
Could you confirm if you have an issue before you add extra blocks to the list?

**Comment by IsaiahJM97** [comment]
Well like I said, that is an issue there and I hadn't added anything to the list. I just noticed that when I turned on things like Iron ore, and Gold ore it became apparent what happened. The blocks that are on the list were there as soon as I installed the mod and loaded it up.
I also apologize for my late replies, I have been somewhat busy recently. I can try again right now as I load up the game and see what happens. I will edit the post with the pictures if possible.
